config/config.toml

```diff
@@ -49,8 +49,10 @@ down_time=false
 tax_processor=true
 transaction_view=false
 x_feature_route=false
+tenant_user=true
```

Comment:
by default should be true is it? Like should be false I guess just check once

Reply:
we should keep it as true once we deploy these changes